mobile/src/home-view.js

```diff
@@ -51,11 +51,11 @@ class HomeView extends PureComponent {
   }
 
   render() {
-    const { suggestedTitle, offerId } = this.props
+    const { suggestedTitle, offersId } = this.props
```

Comment:
What does `offersId` specify?  It makes it sound like a single ID for multiple offers.